meta/alpdesk/alpdesk-core/en.yml

```diff
@@ -0,0 +1,9 @@
+en:
+    title: Alpdesk-Core for Contao
+    description: >
+        JSON based REST Endpoint with JWT Authentication - Extendable by plugins
```

Comment:
If you write `endpoint` in lower case, you probably don't need to add it to the dictionary whitelist?

Reply:
Done in next commit

Reply:
Also removed Alpdesk from de/en because is still in default